packages/flutter_tools/lib/src/android/native_assets.dart

```diff
@@ -4,8 +4,10 @@
 
 import 'package:native_assets_builder/native_assets_builder.dart'
     show BuildResult, DryRunResult;
-import 'package:native_assets_cli/native_assets_cli.dart' hide BuildMode;
-import 'package:native_assets_cli/native_assets_cli.dart' as native_assets_cli;
+import 'package:native_assets_cli/native_assets_cli_internal.dart'
```

Comment:
does `native_assets_cli_internal` mean "should not be imported by anyone else (except package:flutter_tools)"?

Reply:
Not be imported by anyone else except for package:native_assets_builder (shared between dartdev and flutter_tools), dartdev, and flutter_tools.